### ISSUE TITLE
Workflows are denied by webhook "vauth.kb.io"

### ISSUE THREAD

**Issue by STRRL** [chaos-mesh/chaos-mesh#1679]
## Bug Report

`Workflow` is not a type of Chaos, so it does not implement the interface `ChaosValidator`. So all the request about workflow will fail because of that webhook:
```
$ k apply -f ./examples/workflow/workflow.yaml                 
Error from server: error when creating "./examples/workflow/workflow.yaml": admission webhook "vauth.kb.io" denied the request: kind Workflow is not support
```

The current implementation about auth depends on the `SelectSpec`, which is not so well.

I think we need another exactly interface to describe the permissions, like:

```golang
type PermissionScope interface {
	ClusterScoped() bool
	AffectedNamespaces() []string
}
```

Each type of chaos could persist the current implementation, and workflow will always return `false` with `ClusterScoped()` and return it's namespace with `AffectedNamespaces`.

**Comment by STRRL** [comment]
I think we could bypass this issue by just allowing all the requests about workflow and workflow nodes.

**Comment by STRRL** [comment]
closed by #1681
